.github/version/versions.txt

```diff
@@ -1 +1 @@
-0.48.23
```

Comment:
We need to manually skip 0.48.24 because I accidentally generated the build from feature branch - arm-build2. We need to always fix the version scheme if we want to generate the build from feature branch.